docs/reference/plugin/google-analytics.md

```diff
@@ -55,3 +55,24 @@ module.exports = {
   ],
 }
 ```
+
+### debug
+
+- Type: `boolean`
+
+- Details:
+
+  Set to true to enable sending events to DebugView in development.
+
```

Comment:
It would be nice to add a `- Also see: ` and add the link to the debug mode docs.